### ISSUE TITLE
Ioncore Direction - Objective/s

### ISSUE THREAD

**Issue by korvasc** [ioncoincore/ion#17]
**1. Objective** 
2. Strategy
3. Tactics
4. Resource

Adam & Cryptolinux are the visionaries. We need to nail down their vision for IONcore so we can define meaningful objectives. 

**1.Objective**
Adam & Cryptolinux can you start the discussion by sharing your IONcore vision/direction. Then collectively we can define, agree then set IONcore objectives. I've set a deadline of a week for you to both to respond. 

Note once agreed objectives are not set in stone and will probably change as we define the strategy, tactics & resource needed. This is about setting the right direction & expectations.

**Comment by cryptolinux** [comment]
I think this is not really easy to discuss in 1 issue, as every single part requires a separate issue/project to be dealt with, especially as they all are connected, then, you need to actually set priorities too, like:

1. Resources (human as well as financial, as both are own topic for itself, where human does highly depend on financial one, but not the way around).
1.1 Strategy
1.2 Tacticts
1.3 Required human ressources

Where, before getting to point 1.3 we must actually find out the workflow as well as how do we manage and evaluate/audit it. Evaluation as well as audit are two separate processes and if we get into bureaucracy, then there will be almost no end, we need to have to plans, one is where we want to go (kinda mid/long term) as well as how do we deal now with it (short term) to get our long time plan working.

**Comment by korvasc** [comment]
I agree with you, this level of detail will come. A top-down approach helps ensure what limited resource we have isn't wasted. You need to have at least some idea what 1.0 (the objective) is before you try to address the other points. 1.1, 1.2, 1.3.....
